src/vue/pages/VerificationGeneral.vue

```diff
@@ -287,26 +295,26 @@ export default {
           lastName: kycData.last_name,
         },
         documents: {
-          avatar: kycData.documents[DOCUMENT_TYPES.kycAvatar]
-            ? this.wrapDocument(kycData.documents[DOCUMENT_TYPES.kycAvatar])
-            : {},
-          idDocument: kycData.documents[DOCUMENT_TYPES.kycIdDocument]
-            ? this.wrapDocument(kycData.documents[DOCUMENT_TYPES.kycIdDocument])
-            : {},
-          verificationPhoto: kycData.documents[DOCUMENT_TYPES.kycSelfie]
-            ? this.wrapDocument(kycData.documents[DOCUMENT_TYPES.kycSelfie])
-            : {},
+          avatar: _get(kycData, `documents.${DOCUMENT_TYPES.kycAvatar}.key`)
+            ? new DocumentContainer(kycData.documents[DOCUMENT_TYPES.kycAvatar])
+            : null,
```

Comment:
let `wrapDocument` accept the type and do all the deriving document/checking stuff by itself

Reply:
ok